scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/definition/component/parameter/defaults/ConfigParameterDefaultValueDeterminer.scala

```diff
@@ -2,11 +2,9 @@ package pl.touk.nussknacker.engine.definition.component.parameter.defaults
 
 import pl.touk.nussknacker.engine.graph.expression.Expression
 
-object ConfigParameterDefaultValueDeterminer extends ParameterDefaultValueDeterminer {
+object ConfigParameterDefaultValueDeterminer extends ParameterDefaultValueDeterminer with SimpleLanguageDeterminer {
 
-  override def determineParameterDefaultValue(parameters: DefaultValueDeterminerParameters): Option[Expression] = {
-    // TODO: make language configurable as well
```

Comment:
this TODO wasn't realized, just deleted, are we sure we don't want to keep it?

Reply:
Mixed feelieng here, as far as I understand the logic, Editor is enough to determine the language of the expression. I would like to avoid situation where we have to define the same thing in several places as it always brings some synchro issues.